dub.sdl

```diff
@@ -13,6 +13,22 @@ configuration "application" {
 	versions "DubUseCurl"
 }
 
+configuration "nonet" {
+	targetType "executable"
+	mainSourceFile "source/app.d"
+	dependency "vibe-d:http" version=">=0.7.30 <=0.9.0"
+	versions "VibeUseOpenSSL11"
```

Comment:
Since this is target system specific, we shouldn't include this by default. I think this is also the reason for the Travis failure.